### PR TITLE
Ball

### PR DESCRIPTION
**Pull request by BallballTH**
Added:
print hp and sa bar
choose option can now change stats

### CODE REVIEW

File: dialogueMaker/dialogueMaker.cpp [hunk 1]
```diff
@@ -210,3 +214,15 @@ std::string Game::parseText(std::string text) {
...
+void Game::addPlayer(player p){
+  Player = p;
+}
+
+void Game::printstats(){
+  Player.printstats();
+
+}
+
+player Game::Player;
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.cpp]: Necessary?

File: dialogueMaker/dialogueMaker.h [hunk 2]
```diff
@@ -64,4 +69,9 @@ class Game {
     static void printAllScenes();
 
     static void runGame(std::string startSceneId);
+
+    static void addPlayer(player p);
+
+    static void printstats();
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.h]: should be private unless the user wants to print it.

File: tempCodeRunnerFile.cpp
Comment (OrneryCash) [inline tempCodeRunnerFile.cpp:1]: Should be deleted it.

**Review by OrneryCash — CHANGES_REQUESTED** [review]

File: dialogueMaker/dialogueMaker.cpp (hunk 1 above)
Comment (BallballTH) [reply dialogueMaker/dialogueMaker.cpp]: image
this throw up if delete player Game::Player;

File: dialogueMaker/dialogueMaker.h (hunk 2 above)
Comment (BallballTH) [reply dialogueMaker/dialogueMaker.h]: Stats should be displayed all the time, right?

Comment (OrneryCash) [reply dialogueMaker/dialogueMaker.h]: It isn't used in the main.cpp right? It's only used in the dialogueMaker.

File: dialogueMaker/dialogueMaker.h [hunk 3]
```diff
@@ -40,6 +43,7 @@ class Scene {
 };
 
 class Game {
+  static player Player;
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.h]:
```suggestion
  static inline player Player;
```
To solve above issue.

**Review by OrneryCash — CHANGES_REQUESTED** [review]

File: tempCodeRunnerFile.cpp
Comment (OrneryCash) [inline tempCodeRunnerFile.cpp:1]: Be sure to delete this

**Review by OrneryCash — CHANGES_REQUESTED** [review]

**Review by OrneryCash — APPROVED** [review]

**Pull request merged**